javascript/node/selenium-webdriver/lib/virtual_authenticator.js

```diff
@@ -0,0 +1,230 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+'use strict'
+
+/**
+ * Options for the creation of virtual authenticators.
+ * @see http://w3c.github.io/webauthn/#sctn-automation
+ */
+class VirtualAuthenticatorOptions {
+
+  static Protocol = {
+    "CTAP2": 'ctap2',
+    "U2F": 'ctap1/u2f',
```

Comment:
Are we not using `ctap2_1`  ?

Reply:
In Java, it's still `ctap2`
https://github.com/SeleniumHQ/selenium/blob/trunk/java/src/org/openqa/selenium/virtualauthenticator/VirtualAuthenticatorOptions.java#L31

In Python also, it is `ctap2`
https://github.com/SeleniumHQ/selenium/blob/trunk/py/selenium/webdriver/common/virtual_authenticator.py#L36

But according to the official documentation, we are using `ctap2_1`, along with `ctap2`.
https://w3c.github.io/webauthn/#sctn-automation-virtual-authenticators **_(11.3)_**
I followed what Java has implemented.

Reply:
Looking into UI also has two! and i dont see `ctap2_1 `.  may be we can add it in future when required! As all other language bindings has same implementation, we can ignore for now!